common/src/main/java/org/hawkular/accounts/common/AuthServerRequestExecutor.java

```diff
@@ -63,10 +67,12 @@ public String execute(String url, String clientId, String secret, String method)
      */
     public String execute(String url, String urlParameters, String clientId, String secret, String method) throws
             Exception {
+        logger.executingAuthServerRequest(url, urlParameters, clientId, secret, method);
 
         HttpURLConnection connection;
         String credentials = clientId + ":" + secret;
         String authorizationHeader = "Basic " + Base64.getEncoder().encodeToString(credentials.getBytes());
+        logger.authHeaderConvertedTo(clientId, secret, authorizationHeader);
```

Comment:
As I'm logging the encoded auth header (which can be easily decoded back), I'll just not log it then.